NEXT_CHANGELOG.md

```diff
@@ -51,6 +51,42 @@ telemetry:
 By [@bryncooke](https://github.com/bryncooke) in https://github.com/apollographql/router/pull/2338
 
 ## 🚀 Features
+### Configuration upgrades ([Issue #2124](https://github.com/apollographql/router/issues/2124))
+
+The Router will now send anonymous usage telemetry to Apollo that includes a subset of the information listed in the [privacy policy](https://www.apollographql.com/docs/router/privacy/)
+
+It includes information about the environment that the Router is running in, version of the router, the command line args, and configuration shape.
+Note that strings are output as `set` so that we do not leak confidential or sensitive information. 
```

Comment:
I don't understand.  What does `set` mean here?

Reply:
It's from an old version, updated in: [d1cbf56](https://github.com/apollographql/router/pull/2173/commits/d1cbf5675736425b1dbeb9495ef074db2c30e8ce)